tour_model_eval/label_processing.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from scipy.cluster.hierarchy import linkage, dendrogram,fcluster
+
+
+
+# to map the user labels
+# - user_input_df: pass in original user input dataframe, return changed user input dataframe
+# - sp2en: change Spanish to English
+# - cvt_pur_mo: convert purposes and replaced mode
+def map_labels(user_input_df,sp2en,cvt_pur_mo):
+    # Spanish words to English
```

Comment:
My gut  check is to split this into two functions - one for `sp2en` and one for `cvt_pur_mo` instead of smushing them into one giant function. What is the advantage of combining them?

Reply:
changed